src/node-lib/getPageProps.ts

```diff
@@ -62,14 +63,30 @@ const getPageProps = async <PageProps>({
         };
       }
     }
+
+    let oakError = error;
+    /**
+     * If preview mode is on, and error is a ZodError, wrap appropriately
+     */
+    if (context.preview && error instanceof ZodError) {
+      oakError = new OakError({
+        code: "preview/zod-error",
+        originalError: error,
+        meta: {
+          errors: error.errors,
+        },
+      });
+    }
+
```

Comment:
@kimon-satan 
I wonder if instead of trying to reshape the errors here to avoid throwing, if we should add a flag to all error metadata that says if it's in preview mode.

We could then filter them and the alerts in the bugsnag UI so they don't create noise but we still have the error data there for debugging

It also feels like this reshaping into an OakError might be better in the actual CMS layer than `getPageProps`